src/wolff_producer.erl

```diff
@@ -199,11 +200,15 @@ do_init(#{client_id := ClientId,
           partition := Partition,
           config := Config0
          } = St) ->
+  AliasPathSegment = case maps:find(alias, Config0) of
+                         {ok, Alias} when is_binary(Alias) -> Alias;
+                         _ -> Topic
+                     end,
   QCfg = case maps:get(replayq_dir, Config0, false) of
            false ->
              #{mem_only => true};
            BaseDir ->
-             Dir = filename:join([BaseDir, Topic, integer_to_list(Partition)]),
+             Dir = filename:join([BaseDir, AliasPathSegment, integer_to_list(Partition)]),
```

Comment:
should probably AliasPathSegment like what we did here:
https://github.com/emqx/pulsar-client-erl/blob/02ddfa49c24ea205bceb076f7286fcfa7d9261de/src/pulsar_producer.erl#L1098-L1101